integration-tests/deployment/ccip/add_chain.go

```diff
@@ -193,3 +166,105 @@ func NewChainInboundProposal(
 		"0s", // TODO: Should be parameterized.
 	)
 }
+
+func NewSetCandidateProposal(
```

Comment:
Lets add a function comment. Importantly this function:
- Generates a proposal to set candidate OCR instances for both commit. Setting the commit candidate is done through the AddDON call and setting the exec candidate is done through an UpdateDON.